webapp/crashstats/crashstats/static/crashstats/js/socorro/utils.js

```diff
@@ -197,7 +198,8 @@
 
         // Convert the array of strings into an object
         for (i = 0; i < len; i++) {
-          temp = queries[i].split('=');
```

Comment:
When `queries[i]` is something like `date=>%3D2024-01-10`, then we get:

```javascript
temp = ['date', '>%3D2024-01-10']
```

and then everything works super.

The problem scenario here is when `queries[i]` is something like `date=>=2024-01-10`. In that case, we get:

```javascript
temp = ['date', '>', '2024-01-10']
```

Then the `key` becomes `date` and the `value` becomes `>` and the `2024-01-10` part is completely dumped. Then shenanigans ensue and everything goes sideways.

The fix is to only split on the first `=`, but doing `.split('=', 1)` doesn't do what it does in Python--it does this other weird thing which is wildly unhelpful.

So instead, I went with manually splitting the string using `.indexOf('=')` which gives us the index and then creating `temp` by using `slice` to get the key and value as the 0 and 1 indexes.